flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/dialect/DataSourceDialect.java

```diff
@@ -71,6 +71,12 @@ public interface DataSourceDialect<C extends SourceConfig>
     /** The task context used for fetch task to fetch data from external systems. */
     FetchTask.Context createFetchTaskContext(SourceSplitBase sourceSplitBase, C sourceConfig);
 
+    /** Try to reuse Context if reuse is true. */
```

Comment:
I wonder which situation reuse is true , and which situation resuse is false? If always reuse,  no need the param resuse?